static/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -61,6 +66,18 @@ function InviteMembersModal({
     );
   }
 
+  // Render an Alert component based on the error state
+  const renderErrorAlert = () => {
+    if (error) {
+      return (
+        <Alert type="error" showIcon>
+          {error}
+        </Alert>
+      );
+    }
+    return null;
+  };
```

Comment:
also does it need to be a function?
```suggestion
  const errorAlert = error
    ? (
        <Alert type="error" showIcon>
          {error}
        </Alert>
      )
    : null;
```